### PR TITLE
fix(ci): handle concurrent hotfix and release (#3329)

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

**PLEASE MAKE SURE THAT THE BRANCH PR INCLUDES JIRA TICKET ID** (_for centreon-internal_)

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** #MON-35276

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Comment by sc979** [comment]
There's multiple issues to fix ;) 
https://github.com/centreon/centreon/actions/runs/8417958802/job/23047438751?pr=3677#step:4:11

File: .github/actions/promote-to-stable/action.yml [hunk 1]
```diff
@@ -31,14 +40,38 @@ runs:
...
+        echo "[DEBUG] - release_cloud: ${{ inputs.release_cloud }}"
+        echo "[DEBUG] - release_type: ${{ inputs.release_type }}"
 
+        # Cloud specific promote
+        # delivery by default to onprem, override to internal if base branch is master
+        if [[ ${{ inputs.github_base_ref }} == "master" ]]; then
+          ROOT_REPO_PATH="rpm-standard-internal"
+        else
+          ROOT_REPO_PATH="rpm-standard"
+        fi
```
Comment (sc979) [inline .github/actions/promote-to-stable/action.yml:58]:
```suggestion
        ROOT_REPO_PATH="rpm-standard"
        if [[ ${{ inputs.github_base_ref }} == "master" ]]; then
          ROOT_REPO_PATH="rpm-standard-internal"
        fi
```

File: .github/actions/rpm-delivery/action.yml [hunk 2]
```diff
@@ -78,15 +92,19 @@ runs:
         fi
 
         # Deliver based on inputs
-        for ROOT_REPO_PATH in "rpm-standard" "rpm-standard-internal"; do
+        #for ROOT_REPO_PATH in "rpm-standard" "rpm-standard-internal"; do
```
Comment (sc979) [inline .github/actions/rpm-delivery/action.yml:95]:
```suggestion
```

Comment (tuntoja) [reply .github/actions/rpm-delivery/action.yml:95]: I m updating this , since the line 96 needs to be removed as well

**Review by sc979 — DISMISSED** [review]

**Review by paul-oureib — APPROVED** [review]

**Pull request merged**